view/src/test/java/cmm/apps/esmorga/view/screenshot/eventdetails/EventDetailsScreenshotTest.kt

```diff
@@ -0,0 +1,42 @@
+package cmm.apps.esmorga.view.screenshot.eventdetails
+
+import cmm.apps.esmorga.view.eventdetails.EventDetailsView
+import cmm.apps.esmorga.view.eventdetails.model.EventDetailsUiState
+import cmm.apps.esmorga.view.screenshot.BaseScreenshotTest
+import cmm.apps.esmorga.view.theme.EsmorgaTheme
+import org.junit.Test
+
+class EventDetailsScreenshotTest : BaseScreenshotTest() {
+
+    @Test
+    fun eventDetailsView_lightTheme_no_location() {
+        snapshotWithState(lat = null, lng = null)
+    }
+
+    @Test
+    fun eventDetailsView_lightTheme_data() {
+        snapshotWithState()
+    }
+
+    private fun snapshotWithState(lat: Double? = 0.0, lng: Double? = 2.88) {
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/cmm-apps-android/EsmorgaAndroid/pull/5#issuecomment-2194728801>#267d9d</a><span> - 06/27/2024, 01:47 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'snapshotWithState' function sets default values for 'lat' and 'lng' parameters to '0.0' and '2.88' respectively. This could lead to misleading test results if these values are not representative of actual use cases.
</div>
<div id="fix">
<b>Fix:</b> Remove the default values for 'lat' and 'lng' parameters to ensure that the test cases explicitly provide these values, making the tests more reliable and representative of actual scenarios.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -21,7 +21,7 @@
  private fun snapshotWithState(lat: Double?, lng: Double?) {
      paparazzi.snapshot {
          EsmorgaTheme(darkTheme = false) {
              EventDetailsView(
                  uiState = EventDetailsUiState(
                      id = "1",
                      title = "Mobgen fest",
                      subtitle = "35 de Mayo a las 27:00",
                      description = "El mejor evento del año",
                      image = "test.png",
                      locationName = "Mi casa",
                      locationLat = lat,
                      locationLng = lng
                  ),
                  onNavigateClicked = {},
                  onBackPressed = {}
              )
          }
      }
  }
```
</div>
</details>
</div>
<br>


</div>